pkg/controllers/cnset/controller.go

```diff
@@ -118,7 +118,7 @@ func (c *Actor) Observe(ctx *recon.Context[*v1alpha1.CNSet]) (recon.Action[*v1al
 		})
 	}
 	cn.Status.Stores = stores
-	cn.Status.Replicas = cs.Status.Replicas
+	cn.Status.Replicas = cs.Status.ReadyReplicas
```

Comment:
we need add a `ReadyReplicas` field since `Replicas` is for autoscaler

Reply:
ok